src/marqo/tensor_search/health.py

```diff
@@ -152,14 +153,20 @@ def get_marqo_os_status(config: Config, index_name: Optional[str] = None) -> Uni
     else:
         marqo_os_status = HealthStatuses.red
 
-    # Returns red if disk watermark is breached.
-    marqo_os_status = max(marqo_os_status, check_opensearch_disk_watermark_breach(config))
+    marqo_os_disk_watermark_breached = check_opensearch_disk_watermark_breach(config)
+    # Storage is available if disk watermark is not breached (green).
+    if marqo_os_disk_watermark_breached == HealthStatuses.green:
+        marqo_os_storage_is_available = True
+    else:
+        marqo_os_storage_is_available = False
+    # Returns red status if disk watermark is breached.
```

Comment:
We agreed on a non-red status for this. I am not 100% sure whether it is green or yellow. I'll bring this up.

Reply:
Changing to yellow status